docker-gcloud-pubsub-emulator/run.sh

```diff
@@ -5,9 +5,12 @@
 # After it's done, port 8682 will be open to facilitate the wait-for and
 # wait-for-it scripts.
 (
-    set -eu
```

Comment:
I have to get rid of this setting because the script will terminate if PUBSUB_EMULATOR_WAIT_TIMEOUT is not defined. I would get the following error:
```
/run.sh: 11: PUBSUB_EMULATOR_WAIT_TIMEOUT: parameter not set
```
I was trying to make this new env variable optional so that the behavior is the same as before (it waits only 15s).